vcpkg-ports/argtable3/portfile.cmake

```diff
@@ -5,14 +5,13 @@ vcpkg_from_github(
     SHA512 623197142fd1749b2fd5bc3e51758ae49c58ec8699b6afa5ecb2d0199d98f9c05366f92c5169c8039b5c417f4774fb4a09c879a7b04ddbed9d5e43585692ed7f
     HEAD_REF master
     PATCHES
-        "Fix-dependence-getopt.patch"
         "Remove-DllMain.patch"
 )
 
 set(ARGTABLE3_REPLACE_GETOPT ON)
-if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-   set(ARGTABLE3_REPLACE_GETOPT OFF)
-endif()
+#if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+#   set(ARGTABLE3_REPLACE_GETOPT OFF)
+#endif()
```

Comment:
Why comment out instead of removing these?

Reply:
no reason, removed them.